torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -415,7 +414,6 @@ void RequestCallbackImpl::processPythonRRefFetchCall(
       Message m =
           PythonRRefFetchRet(std::move(*result).toIValues()).toMessage();
       m.setId(messageId);
-      rref->blockAllStreams(lsctx);
```

Comment:
`blockAllStreams` enqueues a wait for the RRef's events on each of `lsctx`'s streams. However, a few lines above, we call `getValue` on the RRef which calls `wait` on the future, which already does that.